### ISSUE TITLE
BIP39 recovery for CJK languages is broken.

### ISSUE THREAD

**Issue by junderw** [Electron-Cash/Electron-Cash#2740]
When the electrum mnemonic branched away from BIP39, Thomas added some steps to the normalization of text for the new electrum seed format.

These new steps included stripping whitespace from CJK phrases. (Chinese, Japanese, Korean)

In current Electrum version, BIP39 recovery uses only `unicodedata.normalize('NFKD', ...)` but Electron Cash has switched to using the Electrum mnemonic normalize function for BIP39 as well.

This doesn't affect English seeds, so it's hard to notice.

Example:

とおく　かんち　そうご　まんが　きかく　ほいく　くうふく　てふだ　けわしい　たちばな　ひつよう　ほあん

This is a valid BIP39 phrase. In Electrum, you can enter it (it will say invalid checksum because it can't verify it, but restore it anyways) and it will show the same addresses as on https://iancoleman.io/bip39/

However, if you put it into Electron Cash, it will strip the spaces, and create a different seed. (And it will even show the space-stripped version when you try to view the seed in the wallet > seed window.

So in summary, using the normalize_text function used in Electrum seeds everywhere is not correct. When normalizing a BIP39 phrase it must only be NFKD normalized and nothing else.
